studio/src/rte/ost.js

```diff
@@ -174,7 +190,7 @@ export function getOffferSelectorTool() {
     `;
 }
 
-export function openOfferSelectorTool(offerElement) {
+export function openOfferSelectorTool(triggerElement, offerElement) {
     const landscape =
```

Comment:
we should probably add a `try/catch` in open and close methods as they becomes more complex.